README.md

```diff
@@ -1,5 +1,4 @@
-macOS Cookbook
-==============
+# macOS Cookbook
```

Comment:
Why was the markdown changed here? I think the syntax that was there previously makes the raw file more readable, and it's the same functionally.

Reply:
These changes were from `markdownlint`